nalgebra-sparse/Cargo.toml

```diff
@@ -15,6 +15,7 @@ license = "Apache-2.0"
 [features]
 proptest-support = ["proptest", "nalgebra/proptest-support"]
 compare = [ "matrixcompare-core" ]
+io      = [ "pest", "pest_derive" ]
```

Comment:
Should add `io` to the set of features used on `docs.rs` at the bottom of this file